### PR TITLE
Vinai/elixir smoke test

### PR DESCRIPTION
**Pull request by VinaiRachakonda**

### CODE REVIEW

File: integration-tests/MySQLDockerfile [hunk 1]
```diff
@@ -96,4 +103,4 @@ COPY mysql-client-tests /mysql-client-tests
 COPY mysql-client-tests/mysql-client-tests-entrypoint.sh /mysql-client-tests/entrypoint.sh
 
 WORKDIR /mysql-client-tests
-ENTRYPOINT ["/mysql-client-tests/entrypoint.sh"]
+ENTRYPOINT ["/mysql-client-tests/entrypoint.sh"]
```
Comment (coffeegoddd) [inline integration-tests/MySQLDockerfile:106]: newline

File: integration-tests/mysql-client-tests/mysql-client-tests.bats [hunk 2]
```diff
@@ -100,6 +100,17 @@ cmake ..
     ruby $BATS_TEST_DIRNAME/ruby/ruby-mysql-test.rb $USER $PORT $REPO_NAME
 }
 
+@test "elixir myxql test" {
+    cd $BATS_TEST_DIRNAME/elixir/
+    # install some mix dependencies
```
Comment (coffeegoddd) [inline integration-tests/mysql-client-tests/mysql-client-tests.bats:105]: can we do this in the dockerfile with the rest of the deps?

**Review by coffeegoddd — APPROVED** [review]
LG, couple comments

File: integration-tests/mysql-client-tests/mysql-client-tests.bats (hunk 2 above)
Comment (VinaiRachakonda) [reply integration-tests/mysql-client-tests/mysql-client-tests.bats]: I think elixir expects dependencies to be local so I have to leave it like this :/ Tried the other way around and it didn't work

**Pull request merged**